sys/oom_linux.go

```diff
@@ -24,25 +24,42 @@ import (
 	"strings"
 
 	"github.com/containerd/containerd/pkg/userns"
+	"golang.org/x/sys/unix"
 )
 
 const (
 	// OOMScoreMaxKillable is the maximum score keeping the process killable by the oom killer
 	OOMScoreMaxKillable = -999
-	// OOMScoreAdjMax is from OOM_SCORE_ADJ_MAX https://github.com/torvalds/linux/blob/master/include/uapi/linux/oom.h
+	// OOMScoreAdjMax is from OOM_SCORE_ADJ_MAX https://github.com/torvalds/linux/blob/v5.10/include/uapi/linux/oom.h#L10
 	OOMScoreAdjMax = 1000
+	// oomScoreAdjMin is from OOM_SCORE_ADJ_MIN https://github.com/torvalds/linux/blob/v5.10/include/uapi/linux/oom.h#L9
+	oomScoreAdjMin = -1000
```

Comment:
extremely minor, but it looks weird to export the max and not the min; I understand the max can be used elsewhere (although non-test code use of it may now be zero because of other changes in this PR 😄 ). Not sure it matters that much

Reply:
I was considering to un-export OOMScoreAdjMax and have them both private (until needed elsewhere), and because this one was new, didn't want to export it for the same reason; happy to change either way; do you think I should export both, or un-export?

Reply:
Looks like OOMScoreAdjMax was used in a test, so I exported them both